engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
@tdgunes, I just had a thought. If it's too difficult to split the content of RenderableWorldImpl into the nodes that use its acceleration structures, perhaps we can turn it into a node of its own, on which other nodes are dependent.